index.bs

```diff
@@ -18,21 +20,35 @@ url: http://w3c.github.io/requestidlecallback/#dfn-list-of-idle-request-callback
 urlPrefix: https://html.spec.whatwg.org/multipage/
 	urlPrefix: browsers.html
 		type: dfn; text: unit of related similar-origin browsing contexts
+		type: dfn; text: browsing context
+		type: dfn; text: top-level browsing context
+		type: dfn; text: nested browsing contexts
 	urlPrefix: webappapis.html;
 		type: dfn; text: report the exception
 		type: dfn; text: event loop
 	urlPrefix: infrastructure.html;
 		type: dfn; text: rules for parsing dimension values
 		url: #dfn-callback-this-value; type: dfn; text: callback this value
+	urlPrefix: embedded-content.html;
+		url: #the-iframe-element; type: dfn; text: iframe
 urlPrefix: https://heycam.github.io/webidl/
```

Comment:
You can remove this - Bikeshed already knows about the iframe anchor.  (Also, type:dfn is wrong - it's type:element.)